c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -857,6 +862,8 @@ $(NPM_CMD):
 	$(PRERULE_CMD)
 ifeq ($(OPERATING_SYSTEM), LINUX)
 	npm install npm || sleep 20 || npm install npm || sleep 90 || npm install npm
+else
+	touch $@
```

Comment:
Shouldn't we document the fact that Windows users should create a `npm` file in the root folder of their project ? Or is this file automatically created ?